src/vs/workbench/contrib/terminal/browser/links/terminalLinkManager.ts

```diff
@@ -424,3 +438,8 @@ export interface LineColumnInfo {
 	lineNumber: number;
 	columnNumber: number;
 }
+
```

Comment:
todo: is there a way to provide the class name directly?

Reply:
We could have add to the `ILinkProvider` interface to make this more dynamic:

```ts
interface ILinkProvider {
  /** Uniquely identifies the link provider */
  id: string;
}
```